test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Batch/DefaultODataBatchHandlerTest.cs

```diff
@@ -506,6 +542,182 @@ public async Task SendAsync_Works_ForBatchRequestWithInsertedEntityReferencedInA
 
             Assert.NotNull(customer.Orders?.SingleOrDefault(d => d.Id.Equals(orderId)));
         }
+
+        public static TheoryDataSet<Dictionary<string, string>, (string, string, string)> BatchHeadersTestData
```

Comment:
Is this only accessed once? It should probably be set as a (static) readonly field returned from this getter... as it is currently, youu be rebuilding this on each access...

Reply:
VS has recommended that I use an auto property.